src/naturalearth.jl

```diff
@@ -4,33 +4,137 @@
+        Geographic data provided by the https://www.naturalearthdata.com project.
+        Scale: $(srow.SCALE)
+        Entity: $(srow.ENTITY)
+        Variant: $(srow.VARIANT)
+        """,
+        url,
+        Any,
+        post_fetch_method=DataDeps.unpack
+      ))
+      @datadep_str ID
+    catch
+      throw(ErrorException("download failed due to internet and/or server issues"))
+    end
+  end
+end
 
-```julia
-NaturalEarth.get("admin_0_countries", 110)
-NaturalEarth.get("110m_admin_0_countries")
-```
-"""
-function get(args...; kwargs...)
-  table = NE.naturalearth(args...; kwargs...)
-  GeoIO.asgeotable(table)
+function get(scale, entity, variant; kwargs...)
+  path = download(scale, entity, variant)
+
+  # find Shapefile/GeoTIFF file
+  files = readdir(path; join=true)
+  isgeo(f) = last(splitext(f)) ∈ (".shp", ".tif", ".tiff")
+  file = files[findfirst(isgeo, files)]
+
+  GeoIO.load(file; kwargs...)
+end
+
+# -------------
+# USER HELPERS
+# -------------
+
+countries(; scale=10, kwargs...) = get(scale, "Admin 0 – Countries", "countries"; kwargs...)
+
+function borders(; scale=10, variant=:border, kwargs...)
+  variantstr = if variant == :border
+    if scale == 10
+      "land boundaries"
+    elseif scale == 50
+      "land lines"
+    else
+      "country boundaries"
+    end
+  elseif variant == :mapunit
+    "map unit lines"
+  elseif variant == :maritme
+    "maritime indicators"
+  elseif variant == :maritimechn
+    "maritime indicators China supplement"
+  elseif variant == :pacific
+    "Pacific grouping lines"
+  else
+    varianterror()
+  end
+  get(scale, "Admin 0 – Boundary Lines", variantstr; kwargs...)
+end
+
+function states(; scale=10, variant=:states, kwargs...)
+  variantstr = if variant == :states
+    "states and provinces"
+  elseif variant == :ranks
+    if scale == 10
+      "as scale ranks"
+    else
+      "scale ranks"
+    end
+  elseif variant == :nolakes
+    "without large lakes"
+  elseif variant == :borders
+    if scale == 100
+      "boundaries"
+    else
+      "boundary lines"
+    end
+  else
+    varianterror()
+  end
+  get(scale, "Admin 1 – States, Provinces", variantstr; kwargs...)
 end
 
+counties(; scale=10, kwargs...) = get(scale, "Admin 2 – Counties", "counties"; kwargs...)
+
+populatedplaces(; scale=10, kwargs...) = get(scale, "Populated Places", "populated places"; kwargs...)
+
+roads(; scale=10, kwargs...) = get(scale, "Roads", "roads"; kwargs...)
+
+railroads(; scale=10, kwargs...) = get(scale, "Railroads", "railroads"; kwargs...)
+
+airports(; scale=10, kwargs...) = get(scale, "Airports", "airports"; kwargs...)
+
+ports(; scale=10, kwargs...) = get(scale, "Ports", "ports"; kwargs...)
+
+urbanareas(; scale=10, kwargs...) = get(scale, "Urban Areas", "urban areas"; kwargs...)
+
+usparks(; scale=10, kwargs...) = get(scale, "Parks and Protected Lands", "U.S. national parks"; kwargs...)
+
+timezones(; scale=10, kwargs...) = get(scale, "Timezones", "time zones"; kwargs...)
+
```

Comment:
insert section with helper function `varianterror`, it is not part of the public api